andela/ah-leagueOfLegends
# #160747962 Toggle notifications

## What does this PR do?
```
Add notification subscription
Add get single notification
```

## Description of Task to be completed?
```
Users can subscribe and subscribe to notifications
Users can view all notifications whether read or unread
Users can get a single notification and mark it as read

```

## How should this be manually tested?

* checkout the branch `ch-notifications-feedback-160747962`
* run migrations `python manage.py migrate` 
* after running the server signup two users
* user 1 follow user 2
* user 1 unsubscribe from notifications `api/notifications/unsubscribe`
* User 2 create an article
* User 1 check notifications `api/notifications`
* User 1 should have no notifications
* user 1 subscribe for notifications `api/notifications/subscribe`
* User 2 Create an article
* User 1 should receive notifications


## What are the relevant pivotal tracker stories?

[#160747962](https://www.pivotaltracker.com/story/show/160747962)


## Screenshots if available to support PR.

<img width="1280" alt="screen shot 2018-09-27 at 15 18 27" src="https://user-images.githubusercontent.com/7246507/46145946-36696980-c26a-11e8-8907-58335c4fd89e.png">
<img width="1280" alt="screen shot 2018-09-27 at 15 18 38" src="https://user-images.githubusercontent.com/7246507/46145969-46814900-c26a-11e8-89d7-a7136a752734.png">
